apps/ios/Shared/Views/UserSettings/IncognitoHelp.swift

```diff
@@ -0,0 +1,51 @@
+//
+//  IncognitoHelp.swift
+//  SimpleX (iOS)
+//
+//  Created by JRoberts on 22.08.2022.
+//  Copyright © 2022 SimpleX Chat. All rights reserved.
+//
+
+import SwiftUI
+
+struct IncognitoHelp: View {
+    var body: some View {
+        VStack(alignment: .leading, spacing: 8) {
+            Text("Incognito mode")
+                .font(.largeTitle)
+                .padding(.vertical)
+            ScrollView {
+                VStack(alignment: .leading) {
+                    Group {
+                        Text("Turning Incognito mode on allows you to connect to new contacts and join groups without sharing your main profile. When incognito mode is enabled, for each new conversation a new random profile is generated and sent to a person or group you're connecting to.")
+                        Text("When accepting contact requests received via a contact address, you can decide to connect using a random profile by enabling Incognito mode, even if you've created the address itself outside of Incognito mode.")
```

Comment:
```suggestion
                        Text("When accepting contact requests, you can share a random profile by enabling Incognito mode before accepting it.")
```

Reply:
For a received contact request, you can share a random profile by enabling Incognito mode before accepting it.